tools/tpm2_flushcontext.c

```diff
@@ -79,6 +92,19 @@ static tool_rc process_outputs(ESYS_CONTEXT *ectx) {
     /*
      * 1. Outputs that do not require TPM2_CC_<command> dispatch
      */
+    bool is_file_op_success = true;
+    if (ctx.cp_hash_path) {
+        is_file_op_success = files_save_digest(&ctx.cp_hash, ctx.cp_hash_path);
```

Comment:
This is only used in the scope of the if statement, so just do:
```C
if (ctx.cp_hash_path) {
  bool is_file_op_success = files_save_digest();
  if (!is_file_op_success) {
    return tool_rc_general_error;
  }
}
```
And delete line 95 and make it an empty line.